src/index.js

```diff
@@ -140,6 +140,80 @@
                             });
                         });
                 },
+                putUser (username, metadata = {}, db = databases[ defaultDB ]) {
+                    return db
+                        .putUser(username, {
+                            metadata
+                        })
+                        .then(() => {
+                            return vm.$pouch.putUser(username, metadata, db);
+                        })
+                        .catch(error => {
+                            return new Promise(resolve => {
+                                resolve(error);
+                            });
+                        });
+                },
+                deleteUser (username, db = databases[ defaultDB ]) {
+                    return db
+                        .deleteUser(username)
+                        .then(() => {
```

Comment:
Not sure why you want to trigger the same instance function again when the promise is resolved. This is a double call in my opinion

Reply:
you're right, I don't know why, coding too fastly.